.github/workflows/documentation.yml

```diff
@@ -0,0 +1,66 @@
+name: Build documentation
+
+on:
+  schedule:
+    - cron: '0 0 * * SAT' # every Saturday
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout blog
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.USER_TOKEN }}
```

Comment:
why are you using a USER_TOKEN if you're pushing to the current repository? You need to use GITHUB_TOKEN.